pandas/tests/io/test_sql.py

```diff
@@ -2213,8 +2211,6 @@ def test_to_sql_save_index(self):
         self._to_sql_save_index()
 
     def test_transactions(self):
-        if PY36:
-            pytest.skip("not working on python > 3.5")
```

Comment:
can we say anything about what it would take to fix this?  should it be an xfail?

Reply:
Not sure I follow, those two lines were removed.